source/server/config_validation/server.h

```diff
@@ -24,11 +24,13 @@
 #include "source/common/runtime/runtime_impl.h"
 #include "source/common/secret/secret_manager_impl.h"
 #include "source/common/thread_local/thread_local_impl.h"
+
+// TODO(alyssawilk) address
+#include "source/extensions/listener_managers/listener_manager/listener_manager_impl.h"
```

Comment:
Sill this compile without that extension compiled in? Ideally, config validation would work without the listener manager for all non-listener config.

Reply:
no this forces the listener extension to be compiled, which is why I have a TODO to fix.